spec/installation_spec.sh

```
@@ -266,6 +269,152 @@ fDescribe 'bin/install.e-bash.sh'
+      The output should include "=== operation: ROLLBACK ==="
+      The error should include "Error: Previous version commit no longer exists"
+    End
+
+    It 'should accept valid commit hash'
+      # Create a commit so we have a valid hash
+      touch test-file.txt
+      git add test-file.txt
+      git commit --no-gpg-sign -m "test commit" -q 2>/dev/null || git commit -m "test commit" -q
+
+      # Get a valid commit hash
+      local valid_hash=$(git rev-parse HEAD)
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)